### PR TITLE
Description display

### PR DESCRIPTION
**Pull request by JulianWielga**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable i18next/no-literal-string */
+import "ace-builds/src-noconflict/ace";
```
Comment (Dzuming) [inline designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx:2]: 1. Is it a specific vase to use this variant of this file? If i understand correctly it needs to resolve specific issue https://stackoverflow.com/a/77234551
2. Can we use the minified version of this file `ace-builds/src-min-noconflict/ace`

File: designer/client/src/components/toolbarComponents/Grid9.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Box, styled } from "@mui/material";
+
+export const Grid9 = styled(Box)({
```
Comment (Dzuming) [inline designer/client/src/components/toolbarComponents/Grid9.tsx:3]: Typo? Grid9 -> Grid

**Review by Dzuming — COMMENTED** [review]
I left small comments but generally Looks good to me

File: designer/client/src/components/graph/node-modal/properties.tsx [hunk 3]
```diff
@@ -65,6 +68,17 @@ export function Properties({
...
+            <NodeField
+                isEditMode={isEditMode}
+                showValidation={showValidation}
+                node={node}
+                renderFieldLabel={renderFieldLabel}
+                setProperty={setProperty}
+                errors={errors}
+                fieldType={FieldType.checkbox}
+                fieldName={"additionalFields.showDescription"}
+                description={"Show description every time"}
```
Comment (Dzuming) [inline designer/client/src/components/graph/node-modal/properties.tsx:80]: Version from mockups sounds more meaningful https://www.figma.com/design/eOiVhy9jX6DLYlG7p6IOFu/Nu-Product-Design?node-id=2715-30367&t=1KZBs5rTVypEthi6-0. Can we adjust this text?

File: designer/client/src/components/toolbarComponents/Grid9.tsx (hunk 2 above)
Comment (JulianWielga) [reply designer/client/src/components/toolbarComponents/Grid9.tsx:3]: nope, grid of nine areas.

File: designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx (hunk 1 above)
Comment (JulianWielga) [reply designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx:2]: webpack minimizes it anyway

File: designer/client/src/components/toolbarComponents/Grid9.tsx (hunk 2 above)
Comment (Dzuming) [reply designer/client/src/components/toolbarComponents/Grid9.tsx:3]: Ahh, that makes sense :D

File: designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx (hunk 1 above)
Comment (Dzuming) [reply designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx:2]: What about point 1 ? Do you know, if we have a specific issue that we need to use this version of ace-builds

> webpack minimizes it anyway

Yes you are right, but in the production build only, but generally, it doesn't matter, so we can just leave it.

**Review by Dzuming — APPROVED** [review]

**Review by arkadius — APPROVED** [review]
Backend side realizes what was intended

File: designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx (hunk 1 above)
Comment (JulianWielga) [reply designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx:2]: not changing anything except the order in the import tree. just same as others. now I moved it next to others in ace.ts

**Pull request merged**